clang/test/Sema/c2x-nodiscard.c

```diff
@@ -54,3 +54,9 @@ void test_missiles(void) {
   launch_missiles();
 }
 
+[[nodiscard]] int f3();
+
+void f4() {
```

Comment:
```suggestion
void GH104391(void) {
```

Reply:
@cor3ntin I've updated the test. Thanks for the review.

Reply:
@cor3ntin could you merge it once the CI passes? Thanks